controllers/save_music.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../modules/file_handle'
+require_relative '../models/music_album'
+
+class MusicController
+  include HandleFiles
+
+  def list
+    HandleFiles.read('musics.json').map do |music|
+      obj = MusicAlbum.new(music['album'], music['genres'], music['genre_name'])
+      obj
+    end
+  end
+
+  def save(musics)
+    save_obj = musics.map do |album|
+      { id: album.id, genre: album.genre.name, publish_date: album.publish_date, on_spotify: album.on_spotify }
```

Comment:
## Functionality not implemented as expected
- The functionality has been implemented correctly but never used on the app quit. That's why data is not getting **preserved** in the `musics.json` file on app quit, i.e **pressing key 13**. 
- You can refer the **movies** module and take an idea.